### PR TITLE
Add cross compile to aarch64, armhf, and armel

### PR DESCRIPTION
**Pull request by abalmos**
Needed for nestybox/sysbox#360

### CODE REVIEW

**Comment by abalmos** [comment]
Force push was due to missing signoffs

File: Makefile [hunk 1]
```diff
@@ -79,22 +94,26 @@ RUN_TEST_CONT := $(CONTAINER_ENGINE) run ${DOCKER_RUN_PROXY} \
...
 
-.DEFAULT: $(RUNC_TARGET)
+.DEFAULT: sysbox-runc
+
+$(RUNC_BUILDDIR)/$(RUNC_TARGET): $(SOURCES) $(SYSIPC_SRC) $(LIBSECCOMP_SRC)
+	$(GO_BUILD) -o $(RUNC_BUILDDIR)/$(RUNC_TARGET) .
 
-$(RUNC_TARGET): $(SOURCES) $(SYSIPC_SRC) $(LIBSECCOMP_SRC)
-	$(GO_BUILD) -o $(RUNC_TARGET) .
+sysbox-runc: $(RUNC_BUILDDIR)/$(RUNC_TARGET)
```
Comment (rodnymolina) [inline Makefile:102]: Flip order of these target pairs (see comment in sysbox-fs review).

File: spec.go [hunk 2]
```diff
@@ -142,10 +142,10 @@ func parseIDMap(idMap string, uid, gid, size *uint32) error {
 	*gid = uint32(num[1])
 	*size = uint32(num[2])
 
-	if *uid > math.MaxUint32 || *gid > math.MaxUint32 || *size < syscont.IdRangeMin {
-		return fmt.Errorf("invalid id-map \"%v\": uid and gid must be <= %v, size must be >= %v",
-			idMap, math.MaxUint32, syscont.IdRangeMin)
-	}
+//	if *uid > math.MaxUint32 || *gid > math.MaxUint32 || *size < syscont.IdRangeMin {
```
Comment (rodnymolina) [inline spec.go:145]: Why is this commented?

**Review by rodnymolina — CHANGES_REQUESTED** [review]
Have one question and one suggestion here.

File: spec.go (hunk 2 above)
Comment (abalmos) [reply spec.go:145]: Good catch, I forgot to go back to to this.

This block fails to compile for 32bit. Something about `math.MaxUint32` being too large for `%v`. A quick Google indicated that `%v` should basically be able to print any type, so I'm not sure how to resolve the error--not being a go programmer myself.

Any thoughts?

Comment (rodnymolina) [reply spec.go:145]: Ah, strange, `math.MaxUint32` is just a regular `1<<32 - 1`, please try `%d` instead.

Comment (abalmos) [reply spec.go]: I was able to resolve this compile error using the advise here: https://stackoverflow.com/questions/36231049/why-does-printing-out-max-int-cause-compile-error-in-golang

**Review by rodnymolina — APPROVED** [review]
LGTM

**Pull request merged**